src/helics/cpp98/ValueFederate.hpp

```diff
@@ -433,13 +433,20 @@ class ValueFederate: public virtual Federate {
         return registerGlobalInput(indexed_name, type, units);
     }
 
-    /** get an input by index*/
+    /** get an input by name*/
     Input getInput(const std::string& name)
     {
         return Input(helicsFederateGetInput(fed, name.c_str(), hThrowOnError()));
     }
-    /** get a subscription by index*/
-    Input getSubscription(int index)
+
+    /** get an input by target*/
+    Input getInputByTarget(const std::string& name)
```

Comment:
Unless "name" here is synonymous with "target" I would propose we change the variable name to "target".